docs/ibc/relayer.md

```diff
@@ -0,0 +1,41 @@
+<!--
+order: 4
+-->
+
+# Relayer
```

Comment:
An overview of the relayer process (i.e  why is it necessary, what does it do, and how) would be nice

Reply:
I was thinking this could be done in a later pr that closes the relayer docs issue. I just wanted to add specific discussion around event handling

Reply:
maybe a link to the relayer? https://github.com/iqlusioninc/relayer